### DISCUSSION TITLE
GetBytes Not working, just hanging

### DISCUSSION THREAD

**Discussion by ghost** [omansak/libvideo#218] [Q&A]
I just found this package, and attempted to use it by creating a new app in VS.
When debugging, the app just hangs on the GetBytes method.

Any idea why or how to fix?

the following simply doesn't work for me:

`var youTube = YouTube.Default; // starting point for YouTube actions
    var video = youTube.GetVideo(link); // gets a Video object with info about the video
    File.WriteAllBytes(@"C:\" + video.FullName, video.GetBytes());`

**Comment by Brick1** [comment]
You should submit it into Issues. It seems like nobody was here since you post it.
